src/analyses/libraryFunctions.ml

```diff
@@ -19,6 +19,8 @@ let c_descs_list: (string * LibraryDesc.t) list = LibraryDsl.[
     ("strcpy", special [__ "dest" [w]; __ "src" [r]] @@ fun dest src -> Strcpy { dest; src });
     ("malloc", special [__ "size" []] @@ fun size -> Malloc size);
     ("realloc", special [__ "ptr" [r; f]; __ "size" []] @@ fun ptr size -> Realloc { ptr; size });
+    (* TODO: Do we need accessKind of "r" for "free" or is "f" alone already sufficient? *)
```

Comment:
@sim642: Any opinion on this TODO? You are the one intricately familiar with these invalidation actions!

Reply:
`f` should suffice for `free`. The reason `realloc` also has `r` is because it has to read the existing memory to copy it before it frees.